Controller/RegistrationFOSUser1Controller.php

```diff
@@ -59,16 +58,25 @@ public function registerAction()
                 $route = $this->get('session')->get('sonata_basket_delivery_redirect');
 
                 if (null !== $route) {
+                    // NEXT_MAJOR: remove the if block
+                    @trigger_error(<<<'EOT'
+Setting a redirect url in the sonata_basket_delivery_redirect session variable
+is deprecated since 3.x and will no longer result in a redirection to this url in 4.0.
+EOT
+                    , E_USER_DEPRECATED);
```

Comment:
@core23 : I get a warning from syntastic about "Space found before comma in function call", but php-cs-fixer (and StyleCI) are fine with it, so let's go with that.